CHANGELOG.md

```diff
@@ -1,5 +1,81 @@
 # Change Log
 
+## [0.25.1](https://github.com/confluentinc/ksql/releases/tag/v0.25.1-ksqldb) (2022-04-07)
+
+### Features
+
+* add java client support for push query v2 ALOS through continue method ([#8785](https://github.com/confluentinc/ksql/pull/8785)) ([11ede7c](https://github.com/confluentinc/ksql/commit/11ede7c1fb9169496e9db42459105b98084bdce7))
+* add cleanup service metrics ([#8779](https://github.com/confluentinc/ksql/pull/8779)) ([1cb7846](https://github.com/confluentinc/ksql/commit/1cb7846715d5055e3aad936bcac90b37d51c9b38))
+* add consistency levels to clients ([#8815](https://github.com/confluentinc/ksql/pull/8815)) ([b6a7fab](https://github.com/confluentinc/ksql/commit/b6a7fabea4ef875646ee8dfa21b2ce23359e7822))
+* add physical plan stubs for new query planner ([#8920](https://github.com/confluentinc/ksql/pull/8920)) ([8bb47c9](https://github.com/confluentinc/ksql/commit/8bb47c91808ab11a12bfa3249039b4948423d96b))

```

Comment:
The query planner changes aren't user-facing. I think these should be removed from the changelog. cc @mjsax